src/Stack/Storage/User.hs

```diff
@@ -60,42 +55,6 @@ share [ mkPersist sqlSettings
       , mkMigrate "migrateAll"
     ]
     [persistLowerCase|
-ConfigCacheParent sql="config_cache"
```

Comment:
Does removing the tables lead to any migration errors due to unsafe migrations?

Reply:
No, Persistent simply ignores tables it isn't told of.